tests/moe/test_cute_dsl_fused_moe.py

```
@@ -552,6 +602,126 @@ def test_hook_is_deterministic_across_helper_instances(self):
+        quantized_scale_2d = torch.zeros(128, 1, dtype=torch.uint8)
+        quantized_scale_mma = torch.zeros(32, 4, 1, 4, 1, 1, dtype=torch.uint8)
+        per_token_scale = torch.arange(128, dtype=torch.float32)
+        finalize_calls = []
+
+        def fake_gemm1(**kwargs):
+            assert kwargs["c_dtype"] == "bfloat16"
+            assert kwargs["out_scale"] is None
+            assert kwargs["global_scale"] is None
+            return intermediate_bf16, None
+
+        def fake_quantize(input, global_scale_inv, sf_layout, enable_pdl):
```

Ruff (0.15.15)

[error] Function argument `input` is shadowing a Python builtin

(A002)